### PR TITLE
Added column sort.

### PR DESCRIPTION
**Pull request by TomMuijsers**
The order in which the columns appear seams to depend on the ordering of the raw data set. To control the order of the columns I added a parameter to pass a sorting function for the columns.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "quick-pivot",
-  "version": "2.5.0",
+  "version": "2.5.1",
```
Comment (pat310) [inline package.json]: Can you remove the version number change? I'll update this with `npm`.

**Comment by pat310** [comment]
@TomMuijsers Looks good, just one comment!

File: package.json (hunk 1 above)
Comment (TomMuijsers) [reply package.json]: I discarded the change on the package.json file and force-pushed the updated commit.

**Comment by TomMuijsers** [comment]
@pat310 Could you please review the pull request again?

**Comment by pat310** [comment]
Thanks @TomMuijsers !

**Pull request merged**